### ISSUE TITLE
[Issue] BetterChest crashes with StorageDrawers 

### ISSUE THREAD

**Issue by accessdenied0** [Aroma1997/BetterChests#102]
I have this issue since the BetterChests-1.12.2-3.0.0.0.b7x (70+) versions. 

MC: 1.12.2
Forge: 14.23.1.2571 

If I try to load my instance it crashes and tells me it is missing something. In the loading section you can see that it errors with Storage Drawers.

I apologise if I did something wrong here but it seems to be a compat issue to me.

[Crash Report](https://pastebin.com/RikiiHsA)

**Comment by HellishINC** [comment]
duplicate of #99

**Comment by accessdenied0** [comment]
Uh, 
I'm sorry but your issue is then bad written. I checked your issue before I made this and there was no real clue it is the same issue with Storage Drawers I had. I had to open and search through your report to figure that out.

And additionally, I would recommend to use either the "crash-report" logs in the "crash-reports" folder or the "forge-latest.log" in the "logs" folder. 
The crash-report also list the mods and forge-version you use and which mod one might cause the crash or crashes with.
